blocks/explore-cards/explore-cards.css

```diff
@@ -1 +1,154 @@
-/* TBD */
\ No newline at end of file
+.explore-cards-wrapper {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.explore-cards {
+    width: 100%;
+}
+
+.story-type p {
+    font-size: var(--body-font-size-xs);
+    font-weight: 700;
+    line-height: 13.2px;
+    letter-spacing: 0.06em;
+}
+
+.card-title p {
```

Comment:
I believe the block name is `explore cards`. Please scope every style with parent block name.